packages/kbn-test-subj-selector/test_subj_selector.ts

```diff
@@ -40,25 +42,30 @@ function termToCssSelector(term: string) {
  *   - prefixing a value with `*` will allow matching a `data-test-subj` attribute containing at least one occurrence of value within the string.
  *     - example: `*foo`
  *     - css equivalent: `[data-test-subj*="foo"]`
- *     - DOM match example: <div> data-test-subj="bar-foo" </div>
+ *     - DOM match example: <div data-test-subj="bar-foo"> </div>
+ *
+ *   - prefixing a value with `^` will allow matching a `data-test-subj` attribute beginning with the specified value.
+ *     - example: `^foo`
+ *     - css equivalent: `[data-test-subj^="foo"]`
+ *     - DOM match example: <div data-test-subj="foo_bar"> </div>
  *
  *   - prefixing a value with `~` will allow matching a `data-test-subj` attribute represented as a whitespace-separated list of words, one of which is exactly value
  *     - example: `~foo`
  *     - css equivalent: `[data-test-subj~="foo"]`
- *     - DOM match example: <div> data-test-subj="foo bar" </div>
+ *     - DOM match example: <div data-test-subj="foo bar"> </div>
  *
  *   - the `>` character is used between two values to indicate that the value on the right must match an element inside an element matched by the value on the left
  *     - example: `foo > bar`
  *     - css equivalent: `[data-test-subj=foo] [data-test-subj=bar]`
  *     - DOM match example:
- *       <div> data-test-subj="foo"
- *          <div> data-test-subj="bar" </div>
+ *       <div data-test-subj="foo"
+ *          <div data-test-subj="bar"> </div>
```

Comment:
I think one `>` got lost in the last update. Should probably be
```suggestion
 *       <div data-test-subj="foo">
 *          <div data-test-subj="bar"> </div>
```